contrib/net/http/roundtripper.go

```diff
@@ -66,3 +66,12 @@ func WrapRoundTripper(rt http.RoundTripper, opts ...RoundTripperOption) http.Rou
 		cfg:  cfg,
 	}
 }
+
+// WrapClient modifies the given client's transport to augment it with tracing and returns it.
+func WrapClient(c *http.Client, opts ...RoundTripperOption) *http.Client {
+	if c.Transport == nil {
+		c.Transport = http.DefaultTransport
```

Comment:
Is the `http.DefaultTransport` automatically wrapped?

Reply:
I'm not 100% sure I understand the question. If you're wondering if calling `WrapClient` will accidentally wrap the default transport, the answer is no.

Reply:
Indeed, I misread the code. My question doesn't make much sense.
Everything looks good 👍